packages/utils/src/index.js

```diff
@@ -36,13 +36,20 @@ if (__DEV__) {
 
 export { checkStyles };
 
-export let wrapEvent = (theirHandler, ourHandler) => event => {
+export const wrapEvent = (theirHandler, ourHandler) => event => {
```

Comment:
Just a heads up: any time you make any change to a package (no matter how small) lerna (and our new changelog generator) are going to pick up changes like this and bump the version. So, for example, this will create an entry in the changelog for this PR that bumps both `@reach/slider` *and* `@reach/utils`. Just something to think about.

Reply:
Noted. Reverted!